features/include/pcl/features/integral_image2D.h

```diff
@@ -129,7 +129,7 @@ namespace pcl
 
       /** \brief Destructor */
       virtual
-      ~IntegralImage2D () { }
+      ~IntegralImage2D () = default;
```

Comment:
This class has no virtual functions so it should no have a virtual destructor. Remove it. 

This is an ABI break.